conceptnet5/vectors/transforms.py

```diff
@@ -101,3 +148,62 @@ def make_replacements(small_frame, big_frame, limit=100000):
             if got:
                 replacements[term] = got[0]
     return replacements
+
+
+def choose_small_vocabulary(big_frame):
+    """
+    Choose the vocabulary of the small frame, by eliminating the terms which:
+     - contain more than one word
+     - are not in ConceptNet
+    """
+    DATA = environ.get("CONCEPTNET_BUILD_DATA", "data")
+    concepts = set(line.strip() for line in open(path.join(DATA, 'stats', 'core_concepts.txt')))
+    small_vocab = [term for term in big_frame.index if term.count('_') < 1 and term in concepts]
+    return small_vocab
+
+
+def make_big_frame(frame, lang):
+    """
+     Choose the vocabulary for the big frame and make the big frame. Eliminate the terms which
+     are in languages other than the language specified with the lang parameter.
+    """
+    vocabulary = [term for term in frame.index if get_language(term) == lang]
+    big_frame = frame.ix[vocabulary]
+    return big_frame
+
+
+def make_small_frame(big_frame):
+    """
+    Create a small frame using the output of choose_small_vocabulary()
+    """
+    small_vocab = choose_small_vocabulary(big_frame)
+    return big_frame.ix[small_vocab]
+
+
+def save_replacements(output_filepath, replacements):
+    # Save the replacement dictionary as a mgspack file
+    with open(output_filepath, 'wb') as output_file:
+        msgpack.dump(replacements, output_file)
+
+
+def save_frame(output_filepath, frame):
+    frame.to_hdf(output_filepath, 'mat', encoding='utf-8')
+
+
+@click.command()
+@click.argument('frame-filepath')
+@click.argument('output-dir')
+@click.option('--lang', default='en')
```

Comment:
Just for consistency: I've been using `-l` and `--language` as the ways of specifying the language option in other commands.